api/cephapi/auth/logout.py

```diff
@@ -0,0 +1,9 @@
+from api import Api
+
+
+class Logout(Api):
```

Comment:
Do we really need a class for Logout? This could be a method under auth.py itslef? 
Please correct me if I have got it wrong